docs/contributor_guide/seed_scenarios.rst

```diff
@@ -0,0 +1,85 @@
+
+Debit card fraud investigation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're a data scientist at a Dutch financial services company, and your manager asks you to join an existing team.  This team has deploy a model trained on historical transaction data and now new debit transaction data is arriving.  For each new transaction, the model predicts whether it is potentially fraudulent and then will trigger an alert and inspection by human analysts.  The output that matters for the company is the final decision by the human analyst of whether to block the transaction, allow it but flag for further investigation by anothe team, or flag the transaction as normal.  False negatives mean clients can't get their money back (eg, in a phishing scheme), while false positives may overwhelm the team of human analysts or disrupt clients making legitimate purchases.
+`Weerts et al. (2019) <https://arxiv.org/abs/1907.03334>`_
+
+Measuring brand sentiment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're a member of a team trying to measure brand sentiment from online comments and reviews.  The team hopes to use an existing language model, a third-party service for flagging abusive comments, and then train a more targeted sentiment classifier for your brand on top.
+`Hutchinson et al. (2020) <https://arxiv.org/pdf/2005.00813.pdf>`_
+
+Employment evaluations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A potential client asks you if ML can help with predicting job candidates' suitability for jobs based on a combination of personality tests and body language
+`Rhagavan et al. (2019) <https://arxiv.org/pdf/1906.09208.pdf>`_
+
+Rankings for image search
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're on a team working on improving an image search system after receiving some complaints from users related to fairness.  Users often use this system to find a selection of stock images to use when making multimedia presentations.  In this system, requests start with some information about the context and user creating the query, and your team is trying to incorporate ideas about fairness like diversity and inclusision into how search results are ranked.
+`Mitchell et al. (2020) <https://arxiv.org/pdf/2002.03256.pdf>`_
+
+Sales leads for car loans
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You work at CarCorp, a company that collects special financing data: information on people who need car financing but have either low credit scores or limited credit histories, and sells this data to auto dealer as sales leads.  CarCorp  serves dealers across the United States.  A new project manager asks about leveraging data science to “improve the quality” of leads so that dealers to not churn.  CarCorp has a large amount of historical lead data (2 million leads in 2017 alone), but relatively less data on which leads had been approved for special financing (let alone why the loan was approved).
```

Comment:
We might want to stay well away from this one, given the reputation for predatory terms on such loans (at least in the US).